stk/populations.py

```diff
@@ -328,57 +347,65 @@ def init_from_files(cls, folder, moltype, glob_pattern='*'):
 
     @classmethod
     def init_random(cls,
-                    macromol_class,
+                    constructed_molecule_class,
                     building_blocks,
                     topologies,
                     size):
         """
-        Assembles a population of :class:`.MacroMolecule`.
+        Constructs a population of :class:`.ConstructedMolecule`.
 
         All molecules are held in :attr:`members`.
 
         From the supplied building blocks a random molecule is selected
-        per sublist to for a :class:`.MacroMolecule`. This is done
-        until `size` :class:`.MacroMolecule` have been formed.
+        per sublist to form a :class:`.ConstructedMolecule`. This is
+        done until `size` :class:`.ConstructedMolecule` have been formed.
```

Comment:
line too long. Linter gives many instances of this and over indentation in this file.